todoapp/todos/models.py

```
@@ -1,8 +1,9 @@
 from __future__ import unicode_literals
 
+from django.conf import settings
 from django.db import models
-from django.utils.encoding import smart_text as smart_unicode
-from django.utils.translation import ugettext_lazy as _
+from django.utils.timezone import now
+from django.utils.translation import gettext_lazy as _
```

Ruff (0.12.2)

`django.utils.translation.gettext_lazy` imported but unused

Remove unused import: `django.utils.translation.gettext_lazy`

(F401)